custom_components/tuya_local/devices/poolsana_heatpump.yaml

```diff
@@ -0,0 +1,85 @@
+name: Pool heat pump
+primary_entity:
+  products:
+  - id: abb8kaxcphfzkwsg
+    name: PoolSana InverPower Next
+  entity: climate
+  translation_key: pool_heatpump
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: Cooling
+              value: cool
+            - dps_val: Heating
+              value: heat
+            - dps_val: Auto
+              value: auto
+    - id: 2
+      type: string
+      name: mode
+      hidden: true
+    - id: 4
+      type: integer
+      name: temperature
+      range:
+        min: 70
+        max: 400
+      mapping:
+        - scale: 10
+          step: 10
+    - id: 16
+      type: integer
+      name: current_temperature
+      range:
+        min: -300
+        max: 1000
+      mapping:
+        - scale: 10
+          step: 1
+    - id: 5
+      type: string
+      name: preset_mode
+      mapping:
+        - dps_val: Boost
+          value: boost
+        - dps_val: Silence
+          value: eco
+        - dps_val: Smart
+          value: comfort
+secondary_entities:
+  - entity: sensor
+    class: temperature
+    name: Effluent Temperature
```

Comment:
Effluent makes it sound like sewage, maybe outlet is better.

Reply:
Effluent is the original tuya naming. Because I don't know exactly what sensor this is measuring I didn't want to change naming :-) But if you want I can change

Reply:
Yes, tuya naming is often a bit off due to English not being their native language.  But I am pretty sure this heatpump only pumps out heated water.

Reply:
Yeah so true. I‘ve changed it and hope it is really only pumping water 😉